### PR TITLE
v2.8 release note

### PR DESCRIPTION
**Pull request by QuanluZhang**

### CODE REVIEW

File: docs/source/release.rst [hunk 1]
```diff
@@ -5,6 +5,63 @@
...
+^^^^^^^^^^^^^
+
+* A new translation framework for document
+
+Notable Bugfixes
+^^^^^^^^^^^^^^^^
+
+* Fix TPE import issue for old metrics
+* Fix the issue in TPE nested search space
+* Support `RecursiveScriptModule` in speedup
```
Comment (ultmaster) [inline docs/source/release.rst:62]: Double back-tick

File: docs/source/release.rst [hunk 2]
```diff
@@ -5,6 +5,63 @@
...
+* Support AutoFormer search space in search space hub, thanks our collaborators xxx, xxx
+* Support multi-GPU training of one-shot NAS
+* One-shot NAS supports the NAS API `repeat` and `cell`
+* Refactor of RetiariiExperiment to share the common implementation with HPO experiment
+* CGO supports pytorch-lightning 1.6
+
+Model Compression
+^^^^^^^^^^^^^^^^^
+
+* *Preview* Refactor and improvement of automatic model compress with a new `CompressionExperiment`
```
Comment (ultmaster) [inline docs/source/release.rst:25]: Same here

File: docs/source/release.rst [hunk 3]
```diff
@@ -5,6 +5,63 @@
...
 
+Release 2.8 - 6/16/2022
+-----------------------
+
+Neural Architecture Search
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* Align user experience of one-shot NAS with multi-trial NAS, i.e., users can use one-shot NAS by specifying the corresponding strategy
+* *Preview* Support load/retrain the pre-searched model of some search spaces, i.e., 18 models in 4 different search spaces
+* Support AutoFormer search space in search space hub, thanks our collaborators xxx, xxx
```
Comment (ultmaster) [inline docs/source/release.rst:16]: Fill in blanks.

File: docs/source/release.rst [hunk 4]
```diff
@@ -5,6 +5,63 @@
...
+
+* Make config.tuner.name case insensitive
+* Allow writing configurations of advisor in tuner format, i.e., aligning the configuration of advisor and tuner
+
+Experiment
+^^^^^^^^^^
+
+* Support launching multiple HPO experiments in one process
+* Refactor of the logging mechanism in NNI
+* Refactor of NNI manager globals for flexible and high externsibility
```
Comment (ultmaster) [inline docs/source/release.rst:41]: extensibility

File: docs/source/release.rst [hunk 5]
```diff
@@ -5,6 +5,63 @@
 Change Log
 ==========
 
+Release 2.8 - 6/16/2022
+-----------------------
+
+Neural Architecture Search
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* Align user experience of one-shot NAS with multi-trial NAS, i.e., users can use one-shot NAS by specifying the corresponding strategy
```
Comment (ultmaster) [inline docs/source/release.rst:14]: Suggest adding links to docs, similar to what we did in previous releases.

File: docs/source/release.rst (hunk 1 above)
Comment (QuanluZhang) [reply docs/source/release.rst:62]: fixed

File: docs/source/release.rst [hunk 6]
```diff
@@ -5,6 +5,63 @@
...
+
+* *Preview* Refactor and improvement of automatic model compress with a new `CompressionExperiment`
+* Support customizating module replacement function for unsupported modules in model speedup
+* Support the module replacement function for some user mentioned modules
+* Support output_padding for contransposed2d in model speedup, thanks external contributor xxx
+
+Hyper-Parameter Optimization
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* Make config.tuner.name case insensitive
```
Comment (cruiseliu) [inline docs/source/release.rst:33]: `config.tuner.name` should use inline code style "``"

File: docs/source/release.rst (hunk 4 above)
Comment (cruiseliu) [reply docs/source/release.rst:41]: Suggest to put all internal stuffs into one item.

File: docs/source/release.rst (hunk 6 above)
Comment (QuanluZhang) [reply docs/source/release.rst:33]: updated

File: docs/source/release.rst [hunk 7]
```diff
@@ -5,6 +5,63 @@
...
+WebUI
+^^^^^
+
+* Improve user experience of trial ordering in the overview page
+* Fix the update issue in the trial detail page
+
+Documentation
+^^^^^^^^^^^^^
+
+* A new translation framework for document
```
Comment (J-shang) [inline docs/source/release.rst:55]: We update a new quantization demo in this release, should we put it into release note?
And we should update the date of `New release` `New demo available` in `README.md`, and other places v2.7 -> v2.8

Comment (QuanluZhang) [reply docs/source/release.rst:55]: what is the link of this quantization demo?

Comment (J-shang) [reply docs/source/release.rst:55]: https://www.youtube.com/channel/UCKcafm6861B2mnYhPbZHavw
https://space.bilibili.com/1649051673

https://nni.readthedocs.io/en/latest/tutorials/quantization_quick_start_mnist.html
after we release v2.8 doc:
https://nni.readthedocs.io/en/v2.8/tutorials/quantization_quick_start_mnist.html

Comment (QuanluZhang) [reply docs/source/release.rst:57]: thanks!

File: docs/source/release.rst (hunk 4 above)
Comment (QuanluZhang) [reply docs/source/release.rst]: updated, please check the latest

**Comment by QuanluZhang** [comment]
@ultmaster @liuzhe-lz @J-shang all comments resolved, please review again

File: docs/source/release.rst [hunk 8]
```diff
@@ -5,6 +5,66 @@
...
+* Refactor of RetiariiExperiment to share the common implementation with HPO experiment
+* CGO supports pytorch-lightning 1.6
+
+Model Compression
+^^^^^^^^^^^^^^^^^
+
+* *Preview* Refactor and improvement of automatic model compress with a new ``CompressionExperiment``
+* Support customizating module replacement function for unsupported modules in model speedup
+* Support the module replacement function for some user mentioned modules
+* Support output_padding for contransposed2d in model speedup, thanks external contributor @haoshuai-orka
```
Comment (ultmaster) [inline docs/source/release.rst:28]: convtranspose2d

File: README.md [hunk 9]
```diff
@@ -20,7 +20,7 @@ NNI automates feature engineering, neural architecture search, hyperparameter tu
 
 ## What's NEW! &nbsp;<a href="#nni-released-reminder"><img width="48" src="docs/img/release_icon.png"></a>
 
-* **New release**: [v2.7 is available](https://github.com/microsoft/nni/releases/tag/v2.7) - _released on Apr-18-2022_
+* **New release**: [v2.8 is available](https://github.com/microsoft/nni/releases/tag/v2.8) - _released on Apr-18-2022_
```
Comment (J-shang) [inline README.md:23]: `_released on Apr-18-2022_` -> `_released on June-21-2022_`

File: README.md [hunk 10]
```diff
@@ -20,7 +20,7 @@ NNI automates feature engineering, neural architecture search, hyperparameter tu
...
 * **New demo available**: [Youtube entry](https://www.youtube.com/channel/UCKcafm6861B2mnYhPbZHavw) | [Bilibili 入口](https://space.bilibili.com/1649051673) - _last updated on Apr-18-2022_
```
Comment (J-shang) [inline README.md:24]: `_last updated on Apr-18-2022_` -> `_last updated on June-21-2022_`

**Review by J-shang — APPROVED** [review]

**Review by liuzhe-lz — APPROVED** [review]

**Review by ultmaster — APPROVED** [review]

**Pull request merged**